### DISCUSSION TITLE
Ground Truth

### DISCUSSION THREAD

**Discussion by reversingentropy** [AI-team-UoA/pyJedAI#5] [Q&A] [answered]
Hi, 
I am wondering if it is possible to load two datasets,dataset_1 and dataset_2, without loading a groundtruth dataframe. My assumption is that I don't know whether they are truly these entities. Would this be possible? Thank you!

**Comment by Nikoletos-K** [comment, marked as answer]
Hello,
sorry for the late reply. We are performing now an architecture restructure. By this Friday we hope it will be finished and we will also provide this feature. It is already possible though, but I suggest you to wait until the next release 🚀. 

Thank you for your interest!

**Reply by reversingentropy** [reply]
Thank you!!!

**Reply by Nikoletos-K** [reply]
Here:  https://pyjedai.readthedocs.io/en/latest/tutorials/CleanCleanERwithoutGT.html
